spec/integration/govuk_index/collections_spec.rb

```diff
@@ -32,6 +33,7 @@
     expected_document = {
        "link" => random_example["base_path"],
        "indexable_content" => "Mainstream browse page description",
+       "slug" => random_example["base_path"].gsub(%r{^/browse/}, ''),
```

Comment:
Could this be hard-coded to "benefits"? It's normally better to use explicit values in tests so that you don't accidentally add the same bug to the app code and the test code.

Ditto for the test below.